docs/gno-tooling/cli/gnodev.md

```diff
@@ -0,0 +1,49 @@
+---
+id: gno-tooling-gnodev
+---
+
+# gnodev
+
+Gnodev allows for quick and efficient development of Gno code.
+
+By watching your development directory, gnodev detects changes in your Gno
+code, reflecting them in the state of the node immediately. Gnodev also runs a
+local instance of `gnoweb`, allowing you to see the rendering of your Gno code instantly. 
+
+## Features
+- **In-Memory Node**: Gnodev starts an in-memory node, and automatically loads
+the **examples** folder and any user-specified paths.
+- **Web Interface Server**: Gnodev automatically starts a `gnoweb` server on
+[`localhost:8888`](https://localhost:8888).
+- **Hot Reload**: Gnodev monitors the **examples** folder and any specified for file changes,
+reloading and automatically restarting the node as needed.
+- **State Maintenance**: Gnodev replays all transactions in between reloads,
+ensuring the previous node state is preserved.
+
+## Installation
+Gnodev can be found in the `contribs` folder in the monorepo.
+To install `gnodev`, run `make install`.
+
+## Usage
+Gnodev can be run from anywhere within the monorepo, and it will automatically load the examples
+folder, providing all the packages and realms found in it for use.
+
+![gnodev_usage](../../assets/gno-tooling/gnodev/gnodev.gif)
+
```

Comment:
I don't think we need to specify 'within the repo.' Upon installation, it will remember the folder from which it was installed. It's more accurate to say that you will need to run `gnodev` on the same machine where you installed it. since gnodev depends (for now) ont the `gno` repo, you cannot use `go install gno.land/gnoland/gno/contribs/latest@latest`

Reply:
Changed it up a bit: [7c4db58](https://github.com/gnolang/gno/pull/1599/commits/7c4db58f2cd037f39bad076941d9298f4ef19206)